ycms/cms/forms/bed_assignment/bed_assignment_form.py

```diff
@@ -0,0 +1,46 @@
+import logging
+
+from django import forms
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+
+from ...models import Bed, BedAssignment, MedicalRecord
+from ..custom_model_form import CustomModelForm
```

Comment:
```suggestion
from django import forms
from django.utils import timezone

from ...models import BedAssignment
from ..custom_model_form import CustomModelForm
```
Took me a while to figure out why pylint was passing despite these imports - you are missing an `__init__.py` in the `bed_assignment` directory!